### ISSUE TITLE
rename `Open Workshop` to `Open Instructions` in the top right hamburger menu

### ISSUE THREAD

**Issue by djschny** [vmware-tanzu-labs/educates-training-platform#223]
### Is your feature request related to a problem? Please describe.

The word `workshop` is ambiguous to the average learner that would use educates (or even know they are using it). The word `Terminal` in that list is obvious, but using `Instructions` instead of `Workshop` I believe is much more universal in the context of educational content.

### Describe the solution you'd like

Rename `Open Workshop` to `Open Instructions`

### Describe alternatives you've considered

* `Open Lab Instructions`
* `Open Left Nav`

**Comment by GrahamDumpleton** [comment]
If making a change like this, I would suggest coming up with a way to provide a configuration table to allow localisation of such text. This would allow someone to change such labels to Spanish or Japanese if that is what the workshop content was in. The same could be used to change just the English version of text if desired as the whole dashboard could be used for things other than workshops with instructions so someone may want to change it to something entirely different. There should also perhaps in this configuration be away to disable selectively some menu options, or even add more if people wanted to use it to provide links off to other stuff.

**Comment by djschny** [comment]
Sure that would be nice. However for as simple as the change is, I would suggest getting value with the initial change and a separate change/issue for the more sophisticated logic you mention.

**Comment by GrahamDumpleton** [comment]
As a quick solution to this, add a Javascript file as part of a theme (global or training portal specific) which contains code to select the menu item value and override it. For example, if you were using a global theme defined in the data values file, you would use:

```
websiteStyling:
  workshopDashboard:
    script:  |
      document.querySelector('li#workarea-controls a[data-url="/workshop/"]').innerHTML = "Open Instructions"
```

This should yield the result as shown.

<img width="314" alt="image" src="https://github.com/vmware-tanzu-labs/educates-training-platform/assets/507637/84a66dec-ad83-4098-bbc9-3acf56d36073">

If using secrets for distinct themes, then add the code to the `workshop-dashboard.js` value within the secret.

For details on theme configuration see:

* https://docs.educates.dev/installation-guides/configuration-settings.html?highlight=theme#overriding-styling-of-the-workshop

**Comment by djschny** [comment]
It's a one-line change. I submitted the PR and referenced this issue.

The styling option seems silly for something like this. Not sure what I'm missing.

**Comment by GrahamDumpleton** [comment]
Related to this issue is existing issue https://github.com/vmware-tanzu-labs/educates-training-platform/issues/193 where was asked for pop out icon to be included in top bar of rendered instructions. Was suggested at the time that menu item be removed if this was added. If using custom instructions renderer, would be up to provider of that to provide their own pop out icon.

**Comment by djschny** [comment]
Correct, I saw that, however that didn't talk about removing it from the existing menu. So I was assuming the right `Open Workshop` was going to stay and therefore having this renamed would help there. If #193 should be updated to say that it's removed from the existing menu item, then this is request is not applicable.
